### ISSUE TITLE
[BUG] embedDocuments is undefined

### ISSUE THREAD

**Issue by dxforceiwei** [FlowiseAI/Flowise#996]
Why is 'this.embeddings.embedDocuments(texts)' undefined in the 'addDocuments' method of 'C:\Users\Wiz.Chen\AppData\Roaming\npm\node_modules\flowise\node_modules\langchain\dist\vectorstores\typeorm.cjs'? Consequently, 'vectors' in 'addVectors' is also undefined?"
![AAAA](https://github.com/FlowiseAI/Flowise/assets/22144915/21d71987-06b7-4060-a073-a6145ed9614d)

I uploaded a document, and the document is as follows json file. 
```
{
    "RECORDS": [
        {
            "id": "12634411",
            "EmpID": "A-F0146",
            "WorkDate": "2023/07/09",
            "SWKOnTime": "20:30",
            "SWKOffTime": "05:30",
            "AWKOnTime": "",
            "AWKOffTime": "",
            "TWKOnTime": "",
            "TWKOffTime": "",
            "LeaveType": "",
            "LeaveHr": ".0",
            "LeaveReason": "",
            "OVHr1": ".0",
            "OVHr2": ".0",
            "LateMins": "0",
            "LateTimes": "0",
            "ALateTimes": "0",
            "EarlyMins": "0",
            "EarlyTimes": "0",
            "AEarlyTimes": "0",
            "ABHr": "8.0",
            "HolidayFlag": "Y",
            "EmpType": "3",
            "Cancel": "Y",
            "CrUser": "System",
            "CrDate": "9/7/2023 21:10:00",
            "MdUser": "System_Multi",
            "MdDate": "10/7/2023 09:40:00",
            "OldEmpID": ""
        },
        {
            "id": "12633606",
            "EmpID": "A-F0146",
            "WorkDate": "2023/07/08",
            "SWKOnTime": "20:30",
            "SWKOffTime": "05:30",
            "AWKOnTime": "",
            "AWKOffTime": "",
            "TWKOnTime": "",
            "TWKOffTime": "",
            "LeaveType": "",
            "LeaveHr": ".0",
            "LeaveReason": "",
            "OVHr1": ".0",
            "OVHr2": ".0",
            "LateMins": "0",
            "LateTimes": "0",
            "ALateTimes": "0",
            "EarlyMins": "0",
            "EarlyTimes": "0",
            "AEarlyTimes": "0",
            "ABHr": "8.0",
            "HolidayFlag": "Y",
            "EmpType": "3",
            "Cancel": "Y",
            "CrUser": "System",
            "CrDate": "8/7/2023 21:10:00",
            "MdUser": "System-Daily",
            "MdDate": "9/7/2023 23:46:00",
            "OldEmpID": ""
        }
    ]
}
```

![螢幕擷取畫面 2023-10-02 132338](https://github.com/FlowiseAI/Flowise/assets/22144915/04bee4f7-090f-45b0-a06a-ecb676255ac2)

**Comment by chungyau97** [comment]
Hi @dxforceiwei, I'm not able to replicate your error:
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/dfde0db6-9996-4c03-839d-2fb84de97ffb)

**Comment by dxforceiwei** [comment]
I'm sorry, I found the reason. 
It's because I used the same Deployment Name for both Azure OpenAI Chat and Azure Embedding. After discussing with my team, I learned that there are two distinct Deployment Names: 'gpt-35-turbo-16k-deploy' and 'text-embedding-ada-002-deploy'. I couldn't use just one of them; otherwise, it wouldn't work. I changed Chat to 'gpt-35-turbo-16k-deploy' and Embedding to 'text-embedding-ada-002-deploy' and everything is working fine now! Thanks a lot.

![image](https://github.com/FlowiseAI/Flowise/assets/22144915/66820bdd-d6e1-4b6a-b8c7-03d81a13ce01)
![image](https://github.com/FlowiseAI/Flowise/assets/22144915/a48465bc-9890-41d0-93a7-ef9d9b71d885)
